frontend/src/pages/CourseSelector/courseDescription/CourseDescription.jsx

```diff
@@ -211,50 +228,66 @@ export default function CourseDescription({ structure }) {
                 />
               </Text>
             </Space>
-            <Title level={3} className="text">
-              Requirements
-            </Title>
+              <CollapsibleHeader
```

Comment:
I'm not entirely sure if we should use our own component for collapsible. The antd theme provides a `collapse` [component](https://ant.design/components/collapse/#header) which might be an alternative to this. May be simpler to work with and abit cleaner as it seems like we don't have to worry bout managing states and reinventing the wheel.